### ISSUE TITLE
2.0: Cluster disappeared from UI when hitting back button during cluster provisioning.

### ISSUE THREAD

**Issue by TheKLARKEN** [supergiant/supergiant#776]
Summary of Issue: During the cluster's creation, working with Likitha, we hit "back" on the browser (Safari), and when we tried to go to the home page and find our cluster again, it was gone! We couldn't find it until we restarted the server.

**Steps to Reproduce:**
- [ ] Start the SG Control server.
- [ ] Open the UI.
- [ ] Link an AWS cloud account.
- [ ] Create a cluster with any configuration.
- [ ] Hit "back" in your browser before the cluster is done creating.
- [ ] Go back to "home" and check your clusters.

It should now be gone.

**Expected Results:**
To see the cluster in its beauty when we went "home."

**Actual Results:**
The cluster was ...MIA.

Dev Info: (fill out and add links to log files)

- Go version: `go version go1.9.3 darwin/amd64`
- SG latest commit hash or release tag: `dddedff38267d34e16763fde1955ca4dec1cc5d7`
- Number of Masters and Nodes: `1 and 0`
- Cloud Provider: `AWS`
- Logs from `/tasks/{task_id}/logs/ws` for all tasks spawned clusters: We cannot SSH into the master cause of another issue - that is a new key pair was created though another one was mentioned.
- Log files from your local /tmp folder (these are named as <task>.log):

```
40c-4fcb-af5f-bc1f0265d774.log
time="2018-11-08T14:01:44-06:00" level=info msg="[aws_create_instance] - started"
time="2018-11-08T14:01:44-06:00" level=info msg="[aws_create_instance] - started"
time="2018-11-08T14:01:46-06:00" level=info msg="[aws_create_instance] - using AMI (ID: ami-10eadd6a) Canonical, Ubuntu, 17.10, amd64 artful image build on 2018-01-22"
time="2018-11-08T14:01:47-06:00" level=info msg="[aws_create_instance] - waiting to obtain public IP..."
time="2018-11-08T14:01:48-06:00" level=info msg="[aws_create_instance] - found public ip - 18.204.3.148 for node test-liki-master-1385"
time="2018-11-08T14:01:48-06:00" level=info msg="[aws_create_instance] - success! Created node test-liki-master-1385 with instanceID i-084023bfab1f90a3e"
time="2018-11-08T14:01:48-06:00" level=info msg="[aws_create_instance] - success"
time="2018-11-08T14:01:48-06:00" level=info msg="[ssh] - started"
time="2018-11-08T14:01:48-06:00" level=info msg="[ssh] - success"
time="2018-11-08T14:01:48-06:00" level=info msg="[download_kubernetes_binary] - started"
```

**Comment by TheKLARKEN** [comment]
This also occurs when you reload the page, or otherwise move away from the page.

**Comment by TheKLARKEN** [comment]
Actually, I think this is a symptom of #781 when the server crashes due to `invalid memory address or nil pointer dereference`. 🤔

**Comment by gopherstein** [comment]
This issue appears to be fixed with #889, we are unable to re-create. If this occurs again please reopen the issue.
